helm/fcp-fd-comms/values.yaml

```diff
@@ -22,6 +25,9 @@ port: &port 3039
 container:
   resourceTier: M
   port: *port
+  messageQueueHost: namespace.servicebus.windows.net
+  messagesSubscriptionAddress: fcp-fd-comms
+  messagesTopicAddress: fcp-fd-comms
```

Comment:
If we're changing the env var names anyway, we may as well update these as well.

Reference will need updated in the config map as well so it knows where to pull from.